qiskit/algorithms/gradients/param_shift_sampler_gradient.py

```diff
@@ -30,7 +30,14 @@
 
 
 class ParamShiftSamplerGradient(BaseSamplerGradient):
-    """Compute the gradients of the sampling probability by the parameter shift rule."""
+    """
+    Compute the gradients of the sampling probability by the parameter shift rule [1].
+
+    **Reference:**
+    [1] Wierichs D., Izaac J., Wang C., Lin C.Y.-Y. General parameter-shift rules for quantum
```

Comment:
https://arxiv.org/abs/1811.11184 might be a better reference as they talk in detail about the quantum circuit case which we also implement here 🙂